usr.sbin/fwget/pci/pci_video_amd

```diff
@@ -160,7 +160,7 @@ pci_video_amd()
 			addpkg "gpu-firmware-amd-kmod-sdma-6-0-1"
 			addpkg "gpu-firmware-amd-kmod-vcn-4-0-2"
 			;;
-		0x164e)
+		0x164e|0x13c0)
```

Comment:
This will get matched by the 0x13*) case on line 48, so this change has no effect, I believe.

Reply:
https://github.com/freebsd/freebsd-src/blob/2fa507442cf18e667426e19367a5f1e108354ebf/usr.sbin/fwget/pci/pci_video_amd#L135

But we already have this line? Should that line also be fixed?

Reply:
Yes, that seems like a bug. cc @evadot 

Really, sh switch statements are a poor way to handle this logic. It would be better to use some set of lua tables to map devices to packages. That'd make it easier to variously permit multiple matches, test whether a device has more than one match, etc..